main.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
 
-	_ "github.com/identw/hetzner-cloud-controller-manager/hcloud"
+	_ "github.com/hendrikkiedrowski/hetzner-cloud-controller-manager/hcloud"
```

Comment:
Could you bring the addiction back? I think it's more correct to just change the code in the hcloud folder and add these edits to this PR